bin/test/ci.sh

```diff
@@ -48,13 +48,10 @@ make tests
 
 EXIT_STATUS=$?
 
-pkill iOSDeviceManager
-pkill Simulator
-
 if [ "${EXIT_STATUS}" = "0" ]; then
   echo "Tests passed"
   exit 0
 else
   echo "Tests failed."
   exit 1
-fi
+fi
```

Comment:
Remove all the calabash codesign stuff:  line 17 - 34

Also remove line 36 - 38.